.rubocop.yml

```diff
@@ -0,0 +1,277 @@
+AllCops:
```

Comment:
I didn't intend to commit this file, but we'll roll with it for now since I still want to clean up several things -- these are very close to the default hound rules.